wireguard/umbrel-app.yml

```diff
@@ -0,0 +1,50 @@
+manifestVersion: 1
+id: wireguard
+name: WireGuard
+tagline: Run your own free and unlimited WireGuard VPN
+category: networking
+version: "12"
+port: 38062
```

Comment:
The default Wireguard UI port of 51821 is free right now. No other app uses it, so this can be changed to `51821` so it behaves as users would expect if they were looking at troubleshooting guides on the web.